README.md

```diff
@@ -37,11 +37,15 @@ source 'https://github.com/salemove/glia-ios-podspecs.git'
 user_frameworks!
 
 target 'YOUR_TARGET_NAME' do
-    pod 'GliaWidgets', git: 'https://github.com/salemove/ios-sdk-widgets', tag: 'SDK_VERSION'
+    pod 'GliaWidgets
```

Comment:
Shouldn't this end with a single quote too?

Reply:
you are right, I missed it - cd9d0ef07b75989783783533ed86320e36c413b8